libkmod/docs/meson.build

```diff
@@ -0,0 +1,22 @@
+gnome = import('gnome')
```

Comment:
wondering if we couldn't get away by simply having kernel-doc here. Maybe it would allow more people to have the rendered doc by dropping the gtk-doc dependency.

Anyway, another idea would be to start publishing the docs to https://kmod-project.github.io